netty/src/main/java/io/grpc/netty/NettyServerHandler.java

```diff
@@ -532,7 +532,8 @@ protected void onConnectionError(ChannelHandlerContext ctx, boolean outbound, Th
   @Override
   protected void onStreamError(ChannelHandlerContext ctx, boolean outbound, Throwable cause,
       StreamException http2Ex) {
-    logger.log(Level.WARNING, "Stream Error", cause);
+    Level level = http2Ex.error() == Http2Error.STREAM_CLOSED ? Level.FINE : Level.WARNING;
```

Comment:
Let's use FINE only if `serverStream == null` as well. Really, `connection().stream(Http2Exception.streamId(http2Ex)) == null` is the more precise check, but `serverStream == null` is pretty close and is easy, and I don't think will mislead readers.

Reply:
Do you mean changing the condition to `serverStream == null && http2Ex.error() == Http2Error.STREAM_CLOSED`?

Reply:
Yes, effectively. And the logging lines would need to move slightly. It may be good to use an `if`, but whatever.

Reply:
@ejona86 Cool I have made the suggested change!